web-app/js/campaigns/contest.js

```diff
@@ -21,10 +21,22 @@ $(function () {
         e.preventDefault();
         var $a = $(this);
         console.log($a)
-        $a.parents("ul.nav").find("li").removeClass("active")
-        $a.parent().addClass("active")
-        var listSelector = $a.attr("data-listSelector");
```

Comment:
Try to use const instead var when posible. It reduces scope and reassignment problems

Reply:
Changed the var to const type variables.